flink-kubernetes-operator-autoscaler/src/main/java/org/apache/flink/kubernetes/operator/autoscaler/ScalingMetricCollector.java

```diff
@@ -101,6 +100,7 @@ public CollectedMetricHistory updateMetrics(
             metricHistory.clear();
             metricCollectionStartTs = now;
         }
+        var topology = getJobTopology(flinkService, cr, conf, autoscalerInfo);
```

Comment:
Any reason for moving this?

Reply:
We query than wipe out the topology along with the metrics according to the original logic.